.github/mergeable.yml

```diff
@@ -0,0 +1,269 @@
+    fail:
+      - do: comment
+        payload:
+          body: >
+            In order for us to match your pull request to the correct open source or internal issue, the title should either:
+              - begin with a reference to a Drupal.org issue number in the format of "Issue #issuenumber"
+              - begin with Internal: if it's related to an internal ticket
+              - begin with Updates: if we are updating one of our dependencies
+              - begin with Hotfix: if we are creating a hotfix
+      - do: labels
+        add: 'blocked: drupal.org ticket'
+      - do: checks # default fail case
+        status: 'failure' # Can be: success, failure, neutral, cancelled, timed_out, or action_required
+    pass:
+      - do: labels
+        remove: 'blocked: drupal.org ticket'
+      - do: checks # default fail case
+        status: 'success'
+
+  # Check for meaningful commit messages.
+  - when: pull_request.*
+      name: "Check commit messages"
+      validate:
+        - do: commit
+          message:
+            regex: '^(coding standards|chore|fix|refactor|test|style|perf)(\(\w+\))?:.+$'
+            skip_merge: true
+      fail:
+        - do: comment
+          payload:
+            body: >
+              It would be great if you could create helpful commit messages, maybe a rebase would work before putting it up for review? ;)
+        - do: labels
+          add: 'blocked: commit cleanup'
+        - do: checks # default fail case
+          status: 'failure'
+      pass:
+        - do: labels
+          remove: 'blocked: commit cleanup'
+        - do: checks # default fail case
+          status: 'success'
+
+  # Open Social always maintains multiple releases. We want to make sure that a proper milestone is added.
+  # This is only ignored if PR is internal.
+  - when: pull_request.*
+    name: "Valid milestone"
+    validate:
+      or:
+        - do: milestone
+          no_empty:
+            enabled: true
+        - do: title
+          begins_with:
+            match: [ 'Internal:', 'Hotfix:' ]
+    pass:
+      - do: checks
+        status: 'success'
+    fail:
+      - do: comment
+        payload:
+          body: >
+            A milestone would be super lovely, this way we can map it against a correct release but also be aware
+            of where to cherry-pick. Don't worry if you're unsure what milestone to pick, our maintainers will be
+            able to assist you with that.
+      - do: checks
+        status: 'failure'
+
+  # Let's make sure that the proper labels are added.
+  # See: https://mergeable.readthedocs.io/en/latest/validators/label.html
+  - when: pull_request.*
+    name: "Valid labels"
+    validate:
+      - do: label
+        - must_include:
+            regex: 'team'
+            message: 'Please, add "team" label'
+        - must_include:
+            regex: 'status'
+            message: 'Please, add "status" label'
```

Comment:
Status might also be empty right? If there is no review needed anymore, no backport needed (for a new feature) and work is finished

Reply:
The status can also be don't merge. I will add some filters or I will experiment with `pull_request.*` or `pull_request_review.*